packages/rspack/src/presets/vue.ts

```diff
@@ -8,15 +8,15 @@ export function vue (ctx: RspackConfigContext) {
   const { options, config } = ctx
 
   // @ts-ignore
-  // config.plugins.push(new (VueLoaderPlugin.default || VueLoaderPlugin)())
+  config.plugins.push(new (VueLoaderPlugin.default || VueLoaderPlugin)())
 
   config.module!.rules!.push({
     test: /\.vue$/i,
-    use: ['/Users/daniel/code/nuxt.js/packages/rspack/src/loaders/vue.cjs']
-    // options: {
-    //   reactivityTransform: ctx.nuxt.options.experimental.reactivityTransform,
-    //   ...options.webpack.loaders.vue
-    // }
+    use: require.resolve('vue-rsloader'),
```

Comment:
Here use `vue-rsloader` as a workaround for `vue-loader`, we will support `vue-loader` at furtrue